sdmetrics/reports/utils.py

```diff
@@ -239,9 +262,15 @@ def get_column_plot(real_data, synthetic_data, column_name, metadata):
     if column_name not in synthetic_data.columns:
         raise ValueError(f"Column '{column_name}' not found in synthetic table data.")
 
+    column_meta = columns[column_name]
     sdtype = get_type_from_column_meta(columns[column_name])
-    real_column = real_data[column_name]
-    synthetic_column = synthetic_data[column_name]
+    if sdtype == 'datetime':
+        datetime_format = column_meta.get('format') or column_meta.get('datetime_format')
+        real_column = convert_to_datetime(real_data[column_name], datetime_format)
+        synthetic_column = convert_to_datetime(synthetic_data[column_name], datetime_format)
```

Comment:
This logic seems similar to what's on line 448. Can we have a `convert_datetime_columns` function that takes in the real data column, synthetic data column and column metadata and converts the column if necessary or is the logic too different?